src/core/exception.d

```diff
@@ -315,14 +315,17 @@ unittest
 }
 
 
+deprecated("UnicodeException is now an Error.")
+alias UnicodeException = UnicodeError;
```

Comment:
If we still need `UnicodeException` for validation, then this is wrong, isn't it? It mustn't be an `Error` if it's supposed to be catchable.